packages/client/src/actions/app.ts

```diff
@@ -0,0 +1,29 @@
+import type { App } from '@lens-protocol/graphql';
+import { AppQuery } from '@lens-protocol/graphql';
+import type { ResultAsync } from '@lens-protocol/types';
+
+import type { AppRequest } from '@lens-protocol/graphql';
+import type { AnyClient, SessionClient } from '../clients';
+import type { UnexpectedError } from '../errors';
+
+/**
+ * Fetch an App.
+ *
+ * Using a {@link SessionClient} will yield {@link App#operations}
+ *
+ * ```ts
+ * const result = await fetchApp(anyClient, {
+ *   address?: evmAddress('0xe2f2a5C287993345a840db3B0845fbc70f5935a5'),
```

Comment:
I know this odd syntax that is a mix of types and actual TS was in some of my examples. It was a mistake.

I fixed all occurrences I found. Let's make these examples just with plain working TS.

```suggestion
 *   address: evmAddress('0xe2f2a5C287993345a840db3B0845fbc70f5935a5'),
```